spec/mail/message.mdown

```diff
@@ -259,6 +261,8 @@ The properties of the Message object submitted for creation MUST conform to the
 - **threadId**: This property MUST NOT be included. It is set by the server upon creation.
 - **mailboxIds**: This property MUST be included. The value MUST include the id of either the mailbox with `role == "drafts"` (to save a draft) or the mailbox with `role == "outbox"` (to send the message). If this mailbox does not have `mustBeOnlyMailbox == true`, others may be included too.
 - **inReplyToMessageId**: Optional. If included, the server will look up this message and if found set appropriate `References` and `In-Reply-To` headers. These will override any such headers supplied in the *headers* property. If not found, the creation MUST be rejected with an `inReplyToNotFound` error.
+- **fowardingMessageId**: Optional. If included, the server will look up this message and if found set appropriate `References` and `X-Forwarded-Message-Id` headers. These will override any such headers 
+supplied in the *headers* property. If not found, the creation MUST be rejected with an `fowardingNotFound` error.
```

Comment:
Ok for typo.

It is use at least with thnderbird. I don't know if it is well spread appart this, but we can require the se of this header to make the feature work, no?